readthedocs/core/utils/__init__.py

```diff
@@ -95,7 +95,12 @@ def prepare_build(
         return (None, None)
 
     if not version:
-        version = project.versions.get(slug=LATEST)
+        default_branch = project.get_default_branch()
```

Comment:
We can't use `get_default_branch` here, it is used for other purpose.

Reply:
Ohh.. ok
I have pushed the changes and updated the tests.